addons/storyshots/storyshots-core/src/frameworks/vue/loader.ts

```diff
@@ -19,7 +19,7 @@ function load(options: StoryshotsOptions) {
   const { configPath, config } = options;
   const storybook = require.requireActual('@storybook/vue');
 
-  configure({ configPath, config, storybook });
+  configure({ configPath, ...config, storybook });
```

Comment:
Can be `configure({ ...options, storybook });` as for other framework?

Reply:
✅